pom.xml

```diff
@@ -13,6 +13,7 @@
     <aws-java-sdk.version>1.11.516</aws-java-sdk.version>
     <zipkin-reporter.version>2.8.1</zipkin-reporter.version>
     <spring.version>2.1.3.RELEASE</spring.version>
+    <micrometer.version>1.1.0</micrometer.version>
```

Comment:
Not sure if this is relevant, but Micrometer versions >1.1.1 include fixes for Kafka consumer metrics: https://github.com/micrometer-metrics/micrometer/releases

Reply:
Thanks. Updated to the latest.